widgets/BigQueryTable-batchsink.json

```diff
@@ -47,6 +47,52 @@
             "placeholder": "Google Cloud Storage bucket for temporary data"
           }
         },
+        {
+          "widget-type": "radio-group",
```

Comment:
New plugins should use the 'toggle' type for boolean properties (https://docs.cdap.io/cdap/6.0.0/en/developer-manual/pipelines/developing-plugins/presentation-plugins.html#H2293).

Reply:
Done.